app/commands/training_data/code_tags_sample/normalize_tags.rb

```diff
@@ -5,8 +5,13 @@ class TrainingData::CodeTagsSample::NormalizeTags
 
   def call
     return nil if tags.nil?
+    return nil if normalized_tags.empty?
```

Comment:
I felt that treating empty tags as `nil` makes things a bit easier (e.g. querying)